s3/tests/integration/test-charm-s3/lib/charms/data_platform_libs/v0/s3.py

```diff
@@ -0,0 +1,791 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
How's this file different from `s3_lib.py`?

Reply:
It is the v0. We will update this as well if we copy the content of `s3_lib.py` in the s3 v1 charm lib